<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1199</width>
    <height>845</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="MplCanvas" name="mp_saxs" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>SAXS 1D Plot Setting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_16">
      <item row="0" column="0">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_24">
        <item row="0" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>type:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="pa_type">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <item>
           <property name="text">
            <string>I - q</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>I - log(q)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>log(I) - q</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>log(I) - log(q)</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>offset:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QDoubleSpinBox" name="pa_offset">
          <property name="decimals">
           <number>4</number>
          </property>
          <property name="singleStep">
           <double>0.050000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>normalization:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QComboBox" name="pa_norm">
          <item>
           <property name="text">
            <string>none</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>I' = Iq2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>I' = Iq4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>I' = I/Io</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0" colspan="6">
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>Plot 1D SAXS</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplCanvas</class>
   <extends>QWidget</extends>
   <header>matplot_qt</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
